### PR TITLE
allow save images in multiple sizes

### LINKED ISSUE #2431: How can i save images in multiple sizes
- Laravel Version: 7.2.7
- PHP Version: 5.5.42
- Laravel-admin: 1.5

How can i save images in multiple sizes like
'thumb' => '200x200',
'large'=> '200x200',
'thumb => '200x200',

I have check the documentation no such feature like that.

### PR DESCRIPTION
**Pull request by zeroiszero**
Fixes #2431

Usage:
```
$form->image('photo')
    ->addThumbnailSize('small', 200, 200)
    ->addThumbnailSize('medium', 600, 600);

// To access thumbnail
$model->thumbnail('small', 'photo');
```

Reference: 
- https://laravel-admin.org/docs/#/en/model-form-fields?id=image-upload
- https://docs.laravelvoyager.com/core-concepts/helper-methods

### CODE REVIEW

**Comment by uacode** [comment]
Hello @zeroiszero, add plz, auto width/height http://image.intervention.io/api/widen

->addThumbnailSize('small', 250, 0);

**Comment by zeroiszero** [comment]
@uacode, auto resize width/height added

File: src/Traits/Resizable.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        if (!isset($this->attributes[$attribute])) {
+            return '';
+        }
+
+        // We take image from posts field
+        $image = $this->attributes[$attribute];
+
+        $thumbnail = $this->getThumbnail($image, $type);
+
+        return \Illuminate\Support\Facades\Storage::exists($thumbnail) ? $thumbnail : $image;
```
Comment (uacode) [inline src/Traits/Resizable.php]: I think, need to add disk('admin')->

Comment (zeroiszero) [reply src/Traits/Resizable.php]:
```suggestion
        return \Illuminate\Support\Facades\Storage::Storage::disk(config('admin.upload.disk'))->exists($thumbnail) ? $thumbnail : $image;
```

**Comment by uacode** [comment]
@z-song We need this feature

**Comment by z-song** [comment]
@zeroiszero @uacode I did some code refactoring to make the usage method shorter. 

Now you can use it as follows

```php
$form->image('photo')->thumbnail('small', 200, 200);

// or multiple thumbnails
$form->image('photo')->thumbnail([
    'small'     => [100, 100],
    'medium' => [300, 300],
    'large'      => [600, 600],
]);
```

Thanks for your work!!

**Pull request merged**